### ISSUE TITLE
Fixture data binding

### ISSUE THREAD

**Issue by nevir** [Polymer/test-fixture#2]
It'd be neat to be able to bind data to fixtures too (so we can test `configure` and whatnot):

``` html
<test-fixture id="foo">
  <template>
    <x-foo stuff="{{thing}}"></x-foo>
  </template>
</test-fixture>
```

``` js
foo = fixture('foo', {thing: 123})
```

Or maybe a separate helper, since that's really overloading `fixture`

**Comment by nevir** [comment]
Ooh! or this:

``` js
foo = fixture({foo: {thing, 123}, bar: true})
```

fixture elements as keys, values as objects to bind (or true for regular stamping). or pass an array for easy-mode

**Comment by cdata** [comment]
This sounds great. It will solidify the Polymer dependency, but we can explore a way to tackle this in a "layered" fashion so that at least you have the option to exclude Polymer.

**Comment by nevir** [comment]
Yeah, I think a heuristic of "if Polymer is present, bind via `x-template`" might be good enough

**Comment by cdata** [comment]
Resolved by #4
